hooks.md

```diff
@@ -10,5 +10,5 @@ http://wizard.amigos.institute/ *but should be tracked through bit.ly*
 Hi,
 we are starting first programming school for kids.
 Check out and get in touch!
-Classes will go on also during summer and each class accepts up to 10 kids.
-http://wizard.amigos.institute/ *but should be tracked through bit.ly*
+Classes will go on also during summer and each class accepts up to 10 kids. If parents organise the place in your local school, we are happy to teach kids how to program on your favourite location.
```

Comment:
Isn't it `organize`?
What about something more `juicy` than `teach kids`? ...
We could just come to there place, make some magic happen, meet them there, .... i dont know :-)
What do you think?
Otherwise I like to include that in "HOOKS", but if it is there (because "HOOKS" doesnt actually go to the website, we should include it somewhere else too - maybe under **SCHEDULE** so that parents know about this option to become active in order to get new more favourable "events for them into our "SCHEDULE"